plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseSQLDialect.java

```diff
@@ -82,8 +86,9 @@ public boolean supportsOrderByIndex() {
 
     public void initDriverSettings(JDBCSession session, JDBCDataSource dataSource, JDBCDatabaseMetaData metaData) {
         super.initDriverSettings(session, dataSource, metaData);
-        removeSQLKeyword("DEFAULT");
-        removeSQLKeyword("SYSTEM");
+        for (int i = 0; i < CLICKHOUSE_NONKEYWORDS.length; i++) {
+            removeSQLKeyword(CLICKHOUSE_NONKEYWORDS[i]);
+        }
```

Comment:
Let's use enhances for-loop here.

```suggestion
        for (String word : CLICKHOUSE_NONKEYWORDS) {
            removeSQLKeyword(word);
        }
```